### PR TITLE
add PV monitoring @HLT

### PR DESCRIPTION
**Pull request by mtosi**
backport of PR https://github.com/cms-sw/cmssw/pull/18074

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Comment by mtosi** [comment]
pardon me, but is it related to this PR ?
I think I did not touch the `SLHCUpgradeSimulations/Geometry` package

File: Validation/RecoVertex/src/PrimaryVertexAnalyzer4PUSlimmed.cc [hunk 1]
```diff
@@ -1334,18 +1352,61 @@ void PrimaryVertexAnalyzer4PUSlimmed::analyze(const edm::Event& iEvent,
...
+	    }
+	    ok = false;
+	  }
+	}
+      }
+      if(!ok)
+	continue;
+    }
+
+    {
```
Comment (perrotta) [inline Validation/RecoVertex/src/PrimaryVertexAnalyzer4PUSlimmed.cc]: What below is the exact repetion of what above: cut_and_paste issue, I guess...

**Comment by perrotta** [comment]
please test

**Comment by perrotta** [comment]
+1
Same as #18074 in 91X

**Comment by Martin-Grunewald** [comment]
+1

**Comment by kpedro88** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by civanch** [comment]
+1

**Pull request merged**